src/index.html

```diff
@@ -1,27 +1,485 @@
+        <a class="list list--number-5" data-qa="nav-hover" href="#">
+          Laptops & Computers
+        </a>
+      </li>
+
+      <li class="item item--6">
+        <a class="list" href="#">
+          Gadgets
+        </a>
+      </li>
+
+      <li class="item item--7">
+        <a class="list" href="#">
+          Tablets
+        </a>
+      </li>
+
+      <li class="item item--8">
+        <a class="list" href="#">
+          Photo
+        </a>
+      </li>
+
+      <li class="item item--9">
+        <a class="list" href="#">
+          Video
+        </a>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<main class="main">
+  <section class="product-card" data-qa="card">
+    <div
+      class="product-card__image">
+    </div>
+
+    <div class="product-card__content">
+      <div class="content__text">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="text__code">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="content__mark">
+        <div class="content__mark__stars">
+          <ul class="stars stars--N">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+        </div>
+
+        <div class="mark__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="content__price">
+        <div class="price__text">
+          Price:
+        </div>
+        <div class="price__value">
+          $2,199
+        </div>
+      </div>
+
+      <div class="content__buy">
+        <a href="#" data-qa=«card-hover»>
+          <button class="buy__button">
+            BUY
+          </button>
+        </a>
+      </div>
+    </div>
+  </section>
+  <section class="product-card">
+    <div
+      class="product-card__image">
+    </div>
+
+    <div class="product-card__content">
+      <div class="content__text">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="text__code">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="content__mark">
+        <div class="content__mark__stars">
+          <ul class="stars stars--N">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+        </div>
+
+        <div class="mark__reviews">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="content__price">
+        <div class="price__text">
+          Price:
+        </div>
+        <div class="price__value">
+          $2,199
+        </div>
+      </div>
+
+      <div class="content__buy">
+        <a href="#" data-qa="hover">
```

Comment:
The `data-qa` attribute `hover` is not consistent with other similar attributes. Ensure consistency in naming and usage of `data-qa` attributes across the document.